great-idea-website/css/index.css

```diff
@@ -63,4 +63,156 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+.container {
+	width: 900px;
+	margin: auto;
+	border: 1px solid black;
+}
+
+/* Stretch #1 - fixed nav bar */
+
+.container {
+	background-color: lightgray;
+}
+
+header {
+	position: fixed;
+	width: 900px;
+	top: 0;
+	background-color: teal;
+	z-index: 9999;
+	
+}
+
+header nav {
+	display: inline-block;
+	margin-bottom: 15px;
+	margin-left: 10px;
+}
+
+/* Stretch #2 - new font */
+header nav a {
+	font-family: 'Orbitron', sans-serif;
+	margin: 0 25px 20px 0;
+	color: white;
+	opacity: 0.5;
+	text-decoration: none;
+}
+
+header nav a:hover{
+	opacity: 1.0;
+}
+
+/* Stretch #4a - background image */
+.logo { 
+	display: inline-block;
+	width: 183px;
+	height: 36px;
+	margin-left: 100px;
+	margin-top: 10px;
+	background-image: url("../img/logo.png");
+	background-repeat: no-repeat;
+}
+
+.top-content {
+	display: inline-block;
+	margin-top: 50px;
+	margin-bottom: 30px;
+}
+
+button {
+	display: inline-block;
```

Comment:
To make the button conform more to the design file, you can make the background transparent and add a little box shadow. (_See desktop.jpg in the repo for reference_)